React/package.json

```diff
@@ -46,8 +46,9 @@
     ]
   },
   "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
should we migrate to React+Vite\TS then? Or it will lose all the project structure?

Reply:
technically if TSX files remain the same migration should be smooth because we didn't modify the project structure anywhere (afaik)

Reply:
We can migrate to ```React+Vite``` (if there isn't any issues with devextreme-react) I guess
But let's do it in terms of a separate task :)